src/status_im2/subs/root.cljs

```diff
@@ -268,6 +268,10 @@
 ; media-server
 (reg-root-key-sub :mediaserver/port :mediaserver/port)
 
+; onboarding
+
+(reg-root-key-sub :onboarding-2/profile :onboarding-2/profile)
```

Comment:
this is intentional to have a clear separation of the old onboarding and new onboarding subs. The idea being that when we move away from the old onboarding that we can completely nuke all `onboarding/` subs etc and rename all new `onboarding-2` subs to `onboarding`. 
Sure we could do it now by changing the old to something like `onboarding-old/` instead but this would be a lot of extra changes, noise and work and potentially brings in bugs from missing some parts.
@cammellos and I agreed upon this. 
Can be changed of course but imo it's the better option as is 👍

Reply:
Understood. I did a quick and vague search I haven't seen other `onboarding/` subs or db keys so that's why I don't see necessary to use the `-2` suffix, but it is okay for me anyway